gulp-core-build/src/logging.ts

```diff
@@ -233,7 +238,7 @@ function writeSummary(callback: () => void): void {
         }
 
         if (getWarnings().length) {
-          log('Task warnings:', gutil.colors.yellow(getWarnings().length + '\r\n' + getWarnings().join('\r\n')));
+          log('Task warnings:', gutil.colors.yellow(getWarnings().length.toString()));
```

Comment:
Not officially, but the ts compiler is complaining otherwise. I wonder if it means the bindings for yellow() expect a string